src/modules/Experiences/ExperiencesList/ExperiencesListItem.js

```diff
@@ -23,7 +25,9 @@ const ExperiencesListItem = ({
   selectable,
   classes,
 }) => {
-  const initial = experience.host ? experience.host[0] : '';
+  const initial = experience.host
+    ? experience.host[0].toUpperCase()
+    : experience.title[0].toUpperCase();
```

Comment:
Eventually we need to note somewhere which fields we _must_ receive from the API...since this (and other stuff) will break if `experience.title` is null